test/integration/ipamd/ipamd_suite_test.go

```diff
@@ -34,6 +34,9 @@ var err error
 const (
 	CoreDNSDeploymentName = "coredns"
 	KubeSystemNamespace   = "kube-system"
+	DefaultEniCount       = 1
+	DefaultIpsPerEni      = 15
```

Comment:
Is this hardcoded based on the instance type we are using? Can we instead make this dynamic? Since if we are using locally and have a different instance type then this test will fail.

Reply:
> Is this hardcoded based on the instance type we are using? Can we instead make this dynamic? Since if we are using locally and have a different instance type then this test will fail.

Yes, I assumed we use the same instance type across our tests. But yes I agree, I'll make the values dynamic

Reply:
During release tests, we will be running on our cluster so better to be dynamic. Thanks.

Reply:
@jayanthvn I've updated it to pull dynamically based on the instance type of primary node. 

Assumptions here: 
- Every network adapter has one ENI at the start
- There's no prefix assigned to any interface at the start